readthedocs/doc_builder/python_environments.py

```diff
@@ -224,7 +224,7 @@ def install_core_requirements(self):
             self.project.get_feature_value(
                 Feature.USE_SETUPTOOLS_LATEST,
                 positive='setuptools<40',
-                negative='setuptools==37.0.0',
+                negative='setuptools==39.0.1',
```

Comment:
Similar, any reason not to just pin `<40` here?

Reply:
Just wanted to keep the same pinned status we had before, but I'm fine on pinning `<40` which seems safe.